### ISSUE TITLE
[1.19.2 and older] Connected textures don't work if dynamic resources is manually enabled

### ISSUE THREAD

**Issue by TomatoPuddin** [embeddedt/ModernFix#112]
Environment:
- fabric 1.19.2-0.14.21
- fabric-api 0.76.0
- modernfix 3.6.0
- continuity 2.0.2

config/modernfix-mixins.properties
`mixin.perf.dynamic_resources=true`

config/continuity.json
```json
{
  "connected_textures": true,
  "emissive_textures": true,
  "custom_block_layers": true,
  "use_manual_culling": true
}
```

Resource pack order:
1. continuity/glass_pane_culling_fix
2. continuity/default
3. Fabric Mods
4. vanilla

Screenshot:
![A3(G 85 { RBM%N}R_J_)8X](https://github.com/embeddedt/ModernFix/assets/67496378/5c76fb97-dab7-4291-93a7-5b349655f372)

**Comment by embeddedt** [comment]
Known issue, I'm waiting on the Continuity developer to have some time to work towards a compatibility solution.

**Comment by embeddedt** [comment]
This issue has been fixed for 1.19.4+. For 1.18.2 & 1.19.2 you will need to choose between having connected textures and enabling dynamic resources. A warning is now displayed if both are used.
